src/experimental/safe_html_transform/configure.zcml

```diff
@@ -33,4 +33,9 @@
         handler=".setuphandlers.post_install">
     </genericsetup:importStep>
 
+  <include package="plone.app.registry" />
+  <include package="Products.MimetypesRegistry" />
+  <include package="Products.PortalTransforms" />
+  <include package="Products.PortalTransforms" />
```

Comment:
There are several entirely different dependency mechanisms involved here, don't confuse them.

* Package dependencies in ``setup.py``, that cause python code to be available
* ZCML dependencies in ``configure.zcml``, that cause that package's ZCML code to be loaded when yours is
* GenericSetup dependencies in ``metadata.xml``, that cause that package's GenericSetup profile to be installed whenever yours is installed.

Search for each on docs.plone.org to find out more.

However, the short answer is use the tests. If they still pass, then they weren't necessary.